src/node/db/API.js

```diff
@@ -171,8 +171,12 @@ exports.getText = async (padID, rev) => {
     }
 
     // get the text of this revision
+    // getInternalRevisionAText method returns a text.text and attribs so we ensure we only
+    // return the text - details at https://github.com/ether/etherpad-lite/issues/5073
     const text = await pad.getInternalRevisionAText(rev);
-    return {text};
+    return {
+      text: text.text
+    };
```

Comment:
```suggestion
    const {text} = await pad.getInternalRevisionAText(rev);
    return {text};
```

Reply:
Weird, I thought I took that approach initially but it didn't work for some reason, must have been a brain fart.

Reply:
I prefer leaving the tracked comment in, it's something that in the future might stop me from tripping myself up again.